### DISCUSSION TITLE
Welcome to search_with_ai Discussions!

### DISCUSSION THREAD

**Discussion by yokingma** [yokingma/search_with_ai#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by spammenotinoz** [comment]
Thank-you, this is an absolutely amazing product.
Hint: It also works well with LiteLLM (OpenAI Compatible) so you can map most provider\models to gpt-3.5-turbo, , with FallBack, eg: Groq and they work well.
